package.json

```diff
@@ -61,6 +63,9 @@
     "inquirer": "^9.2.6",
     "open": "^10.1.0",
     "openid-client": "^5.6.5",
+    "picocolors": "1.0.1",
+    "prompts": "2.4.2",
+    "validate-npm-package-name": "5.0.1",
```

Comment:
I suggest maintaining consistency with the way things currently are, and not pin deps (at least not without further discussion and not as part of the scope of this change)

Reply:
That's an argument that can be said for the CLI as well IMO.

My point is - there's quite a discussion to be had on this, and I'd rather we discuss it then introduce inconsistency.

We wouldn't want the CLI to break either, for any reason!

Reply:
I see your point, but the rest of the CLI has pretty good tests — this feature is much harder to test. More importantly, we don't gain any inherent value from consistency here (or in other words, inconsistency doesn't cost anything). I absolutely love consistency but I question what value it brings here.